src/app/test/mocked-objects.ts

```diff
@@ -713,3 +714,19 @@ export const validTool: ExtendedDockstoreTool = {
   registry: DockstoreTool.RegistryEnum.QUAYIO,
   toolname: null
 };
+
+export const mockedNotification: Notification = {
+  id: 123,
+  message: 'TestingTesting123',
+  type: 'SITEWIDE',
+  priority: 'LOW',
+  expiration: null
+};
+
+export const expiredMockNotification: Notification = {
+  id: 121,
+  message: 'Testing123',
+  type: 'SITEWIDE',
+  priority: 'LOW',
+  expiration: new Date('2018-11-25T00:00:00')
```

Comment:
@garyluu Confused by what is going on; when we debugged this with the UI invoking the web service API, the expiration was a number, the timestamp. But the generated Swagger code has the type as a Date. I think for the tests this should be a number, but then the compile complains.

Have you run into this?

Reply:
You can probably do new Date(29301802481203)

Reply:
Doesn't sound too familiar

Reply:
I logged this in #3127